cookiecutter.json

```diff
@@ -1,6 +1,7 @@
 {
     "project_name": "Base Project",
     "project_slug": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}",
+    "project_version": "0.1.0",
```

Comment:
```suggestion
```

I would prefer to leave a default `0.1.0` and let users customize afterwards if needed.